etn-ccis/blui-symbols
# Add two symbols

Fixes an emergency request from Greg's team



## Changes proposed in this Pull Request:

- Added 2 symbols: 
- ATS 300
![image](https://user-images.githubusercontent.com/8997218/227017406-d1a6ee8d-1495-4216-9450-23d29ded53bc.png)
- TC50-100
![image](https://user-images.githubusercontent.com/8997218/227017477-a439f2f4-8cde-44f2-a175-5cf96e458fa9.png)



## To Test:

1. verify that the artifact is present in /symbols folder

2.

``` sh
cd symbols-mui
yarn start
```
